jvm-agent/src/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckBaseMethodVisitor.kt

```diff
@@ -38,7 +39,12 @@ internal open class LincheckBaseMethodVisitor(
     }
 
     protected fun loadNewCodeLocationId(accessPath: AccessPath? = null): Int = adapter.run {
-        val stackTraceElement = StackTraceElement(className, methodName, fileName, lineNumber)
+        val info = metaInfo.smap.getLine("Kotlin", lineNumber)
```

Comment:
The variable name `info` does not reflect what kind of information it stores, please enhance